docs/apis/migration-perm-guidance.md

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
Suggest adding `ms.date` to per [MSDocs guidance required fields](https://review.docs.microsoft.com/en-us/help/contribute/metadata-attributes?branch=master#required-metadata)